src/renderers/native/ReactNativeAttributePayload.js

```diff
@@ -314,6 +318,9 @@ function diffProperties(
   for (var propKey in nextProps) {
     attributeConfig = validAttributes[propKey];
     if (!attributeConfig) {
+      if (__DEV__ && enableStrictAttributeValidation === true && propKey !== 'children') {
+        throw new Error('unsupported attribute: ' + propKey);
```

Comment:
Yeah we should warn and not throw. I have a feeling that it's going to start triggering a ton of warnings on existing react native apps so someone from the team needs to do the work of cleaning up fb codebase before shipping this.